### PR TITLE
fix: tick boundary check breaking liquidity net in direction query

### PR DESCRIPTION
**Pull request by p0mvn**
Closes: #XXX

## What is the purpose of the change

We should be allowing `TickToPrice` conversion of the `MinCurrentTick`. Otherwise, it might break `GetLiquidityNetInDirection` query in some edge cases

## Testing and Verifying

This change is a trivial rework / code cleanup without any test coverage.

## Documentation and Release Note

  - [ ] Does this pull request introduce a new feature or user-facing behavior changes?
  - [ ] Changelog entry added to `Unreleased` section of `CHANGELOG.md`?

Where is the change documented? 
  - [ ] Specification (`x/{module}/README.md`)
  - [ ] Osmosis documentation site
  - [ ] Code comments?
  - [ ] N/A

### CODE REVIEW

File: x/concentrated-liquidity/math/tick.go [hunk 1]
```diff
@@ -74,7 +74,7 @@ func TickToPrice(tickIndex int64) (price sdk.Dec, err error) {
 	geometricExponentIncrementDistanceInTicks := sdkNineDec.Mul(PowTenInternal(-exponentAtPriceOne)).TruncateInt64()
 
 	// Check that the tick index is between min and max value
-	if tickIndex < types.MinInitializedTick {
```
Comment (czarcas7ic) [inline x/concentrated-liquidity/math/tick.go:77]: so to be clear, this means its valid for the tickIndex to be -108000001?

**Comment by p0mvn** [comment]
merged this in another PR
